nuxt.config.js

```diff
@@ -420,6 +420,7 @@ export default defineNuxtConfig({
       rampApiKey: process.env.RAMP_API_KEY,
       transakApiKey: process.env.TRANSAK_API_KEY || '',
       transakEnvironment: process.env.TRANSAK_ENV || 'PRODUCTION',
+      beehiivPublicationId: process.env.BEEHIIV_PUBLICATION_ID,
```

Comment:
so I should put pub id into envs?
done ✅ 
![image](https://github.com/kodadot/nft-gallery/assets/5887929/4e8772fe-3a41-4375-9561-4fb9974997a2)

Reply:
nope, actually you can remove it ,  that was mb forgot to remove it from this repo , https://github.com/kodadot/workers/pull/180 will have the env vars , will need to add BEEHIIV_API_KEY